BDArmory/Weapons/BDExplosivePart.cs

```diff
@@ -318,23 +324,28 @@ public void ParseWarheadType()
                     break;
             }
         }
+
         public void DetonateIfPossible()
+        {
+            DetonateIfPossible(default(Vector3));
+        }
+        public void DetonateIfPossible(Vector3 direction)
```

Comment:
This doesn't work with SI's solution for the `direction`. If SI's solution gives the correct direction for non-"standard" warheads, then I don't think it's necessary to pass `direction` as a parameter.
Similarly for `Detonate` below.

Reply:
Yup, I've changed those, missed them when I was doing the merge, similar for the issue below